util/src/main/java/io/grpc/util/AdvancedTlsX509KeyManager.java

```diff
@@ -131,14 +131,17 @@ public Closeable updateIdentityCredentialsFromFile(File keyFile, File certFile,
       throw new GeneralSecurityException(
           "Files were unmodified before their initial update. Probably a bug.");
     }
+    if (checkNotNull(unit, "unit").toMinutes(period) < MINIMUM_REFRESH_PERIOD) {
+      log.log(Level.INFO,
+          "Provided refresh period of {0} {1} is too small. Default value of {2} minute(s) "
```

Comment:
We never use INFO, because essentially nobody ever wants to see them. Either it is a problem or it isn't, and if it is a problem, throw an exception. We could make this FINE, but I don't think we should have it INFO which is logged by default. Logging at INFO would be the same as saying it is against the API.

Reply:
Oh, I see that it's FINE in a lot of similar places. Thank you, changed.